weave/trace_server/interface/builtin_object_classes/saved_view.py

```diff
@@ -0,0 +1,89 @@
+from typing import Any, Literal, Optional
+
+from pydantic import BaseModel, Field
+
+from weave.trace.traverse import PathElement
+from weave.trace_server.interface.builtin_object_classes import base_object_def
+
+# Compare with definitions in weave/trace_server/trace_server_interface.py
```

Comment:
This looks reasonable - are their any differences between this and the query layer worth calling out? Any reason we did not simply import some of those symbols?

Reply:
Just logically I want us to have the flexibility to make changes to the API layer independently, there may be backwards compatibility issues with the persisted views.